### PR TITLE
move to containerd and crictl

### PR DESCRIPTION
**Pull request by hectorhuertas**

### CODE REVIEW

File: resources/containerd-config.toml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# based on /run/torcx/unpack/docker/usr/share/containerd/config.toml
```
Comment (george-angel) [inline resources/containerd-config.toml]: Is this available online? If yes, could we include a link?

File: resources/install-crictl [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -o errexit
+
+version=$(head -c 5 <<< "${kube_version}").0
+
+wget https://github.com/kubernetes-sigs/cri-tools/releases/download/$${version}/crictl-$${version}-linux-amd64.tar.gz
+sudo tar zxvf crictl-$${version}-linux-amd64.tar.gz -C /opt/bin
```
Comment (george-angel) [inline resources/install-crictl]: Why `sudo`, doesn't this run as root?

Comment (george-angel) [reply resources/install-crictl]: Also, could we do a checksum on the artifact?

1. Its not been mitm meddled with
2. Also check the download was successful

File: resources/containerd-config.toml (hunk 1 above)
Comment (hectorhuertas) [reply resources/containerd-config.toml]: I'll check

**Comment by hectorhuertas** [comment]
We decided not to migrate

https://trello.com/c/KvKf4Btj/708-move-to-containerd-runtime

**Pull request closed without merging**